source/administration/changelog_esr.md

```diff
@@ -0,0 +1,71 @@
+# Mattermost Extended Support Release (ESR) Changelog
```

Comment:
If a separate changelog is created for ESR, should this be linked from the ESR doc?

Reply:
Makes sense. I will add it.